src/controllers/HighlightModeController.js

```diff
@@ -14,9 +21,21 @@ class HighlightModeController extends AnnotationModeController {
      * @return {void}
      */
     handleThreadEvents(thread, data) {
+        let firstAnnotation;
         switch (data.event) {
+            case THREAD_EVENT.save:
+                // Re-render plain highlight canvas when a plain highlight is converted to a highlight comment
+                firstAnnotation = getFirstAnnotation(thread.annotations);
+                if (
```

Comment:
@jeremypress this ensures that when a highlight is converted to a highlight comment, we re-render the canvases so that it is drawn on the correct canvas